bin/cpumask.py

```diff
@@ -16,12 +16,32 @@ class global_vars:
 def process_options():
     '''Define the CLI argument parsing options'''
 
-    parser = argparse.ArgumentParser(description="CPU mask creator")
+    parser = argparse.ArgumentParser(description = "CPU mask creator",
+                                     formatter_class = argparse.ArgumentDefaultsHelpFormatter)
+
+    parser.add_argument("--type",
+                        dest = "type",
+                        help = "What type of input to process",
+                        choices = [ "bitmask", "cpus", "hexmask" ],
+                        default = "cpus")
 
     parser.add_argument("--cpus",
                         dest = "cpus",
-                        help = "Use one or more times to define CPU and/or CPU lists to include in the mask",
+                        help = "Use one or more times to define CPU and/or CPU lists to include",
                         action = "append",
```

Comment:
Currently `--cpus` is using an `append` action so it can be used multiple times and each time it is used another element is added to the list.